.github/workflows/check.yml

```diff
@@ -11,17 +11,17 @@ jobs:
         include:
           - elixir: '1.10.4'
             otp: '23.0.2'
-          - elixir: '1.7.0'
-            otp: '19.0'
+          - elixir: '1.7.4'
```

Comment:
Not sure how other OS packages approach this but it was intentional to test on lowest minor/patch version supported in order to truly ensure that we support Elixir 1.7/OTP 19 regardless of differences that could've been introduced in patches (I know - semver, but still...). Also, supporting newest patch version here adds extra maintenance work.

What do you think?